infra/aws/terraform/s3/k8s-infra-e2e-boskos-scale-001/providers.tf

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
```

Comment:
@pkprzekwas   all resources created here should live in a remote backend. Which backend is used ?

Reply:
Ah, fair point. We don't have any bucket for TF state on this account yet. Let me create and use it.

Reply:
@ameukam added configuration for remote tf state.